docs/get-started/platforms/stm32.md

```diff
@@ -1,4 +1,119 @@
 
 # STM32
 
-TODO
+LVGL Can be added to [STM32CubeIDE](https://www.st.com/en/development-tools/stm32cubeide.html) in a similar fashion to any other Eclipse-based IDE.
+
+## Including LVGL in a Project
+* Create or open a project in STM32CubeIDE.
+* Copy the entire LVGL folder to *[project_folder]/Middlewares/Third_Party/lvgl*.
+* In the STM32CubeIDE **Project Explorer** pane: right click on the LVGL folder that you copied (you may need to refresh the view first before it will appear), and select **Add/remove include path...**. If this doesn't appear, or doesn't work, you can review your project include paths under the **Project** - **Properties** menu, and then navigating to **C/C++ Build** - **Settings** - **Include paths**, and ensuring that the LVGL directory is listed.
+
+Now that the source files are included in your project, follow the instructions for [Porting](https://docs.lvgl.io/master/porting/project.html) your project to create the ```lv_conf.h``` file, and initialise the display.
+
+## Example
```

Comment:
Could you add a bare metal example too? It's super simple
- call `lv_timer_handler()` and `HAL_Delay(5)` in a `while(1)` loop
- Add `lv_tick_inc(1)` to the HAL Tick interrupt handler

I think the best would be to have a "Bare metal"  and a "FreeRTOS" section.